docs/howto/hooks/lua.md

```diff
@@ -373,8 +373,17 @@ Creates a new Delta Lake client used to interact with the lakeFS server.
 ### `formats/delta_client.get_table(repository_id, reference_id, prefix)`
 
 Returns a representation of a Delta Lake table under the given repository, reference, and prefix.
-The format of the response is a table `{number, {string}}` where `number` is a version in the Delta Log, and the mapped 
-`{string}` table (list) contains JSON strings of the different Delta Lake log operations listed in the mapped version entry. 
+The format of the response is two tables: 
+- the first is a table of the format `{number, {string}}` where `number` is a version in the Delta Log, and the mapped `{string}` 
+table (list) contains JSON strings of the different Delta Lake log operations listed in the mapped version entry.
```

Comment:
Sentences are not very clear. table? list?JSON strings?? I'm not sure what I'm getting here

Reply:
it's all true it's a lua [list](https://www.lua.org/pil/11.1.html), but all lists are a flavor of a table. The strings themselves are JSON strings